dev-infrastructure/modules/key-vault-cert.bicep

```diff
@@ -30,7 +30,7 @@ resource newCertwithRotationKV 'Microsoft.Resources/deploymentScripts@2023-08-01
   location: location
   kind: 'AzurePowerShell'
   properties: {
-    azPowerShellVersion: '7.5.0'
+    azPowerShellVersion: '12.0.0'
```

Comment:
that's a pretty large gap, does it work? :sweat_smile:

Reply:
it does. i run the code a lot :)